Fix email padding on staff cards.

jimafisk opened the pull request:
Previously staff cards with different content lengths would have contact sections of different heights:
![staff_orig](https://user-images.githubusercontent.com/5913244/34537229-c7e0a5c4-f095-11e7-8f79-7ea1fc6cb901.png)

This update should make these sections look consistent:
![staff](https://user-images.githubusercontent.com/5913244/34537237-cff8c66a-f095-11e7-866a-01d751d037cd.png)

Fixes https://github.com/CityOfBoston/boston.gov/issues/960

finneganh commented:
TBH I think that the container should also be `flex-direction: column` rather than the default row.

jimafisk commented:
@finneganh you're right, that is a better way to do it and it fixes the broken hover behavior that I didn't catch. I adjusted, squashed, and force pushed.

finneganh commented:
Looks like Percy found some stuff related to the width to take a look at. This would be related to changing the flex direction I expect.

I think that you'll want to have Percy not see a change for this PR.

fionawhim commented on stylesheets/components/card/_person.styl:
```diff
@@ -31,6 +32,8 @@
     background-color: $grey-000
     text-align: left
     width: 100%
+    //height: -webkit-fill-available 
```
remove commented-out

fionawhim commented on stylesheets/components/card/_person.styl:
```diff
@@ -31,6 +32,8 @@
     text-align: left
     width: 100%
+    //height: -webkit-fill-available 
+    height: calc(100% - 50px);
```
This is assuming that the bottom will always only be 50px, which I don't think should be hard-coded.

Should we work through this in person when you're next in the office? I've lost track of what's been suggested so far.

jimafisk replied:
Yeah def, I realized that too and was working on it locally. I had tried setting the bottom to a standard 50px, but that doesn't really work for all displays. I think changing the actual HTML structure of the template like you mentioned might be worth it here. Trying to get adjacent anchor tags to fill up the available space while pass percy is proving difficult.

fionawhim commented:
Note that the goal is not to "pass" Percy in the sense of matching
pixel-for-pixel. The point is to agree that the screenshots Percy gives you
are fine.

On Wed, Jan 17, 2018 at 2:28 PM, Jim Fisk <notifications@github.com> wrote:

> *@jimafisk* commented on this pull request.
> ------------------------------
>
> In stylesheets/components/card/_person.styl
> <https://github.com/CityOfBoston/patterns/pull/211#discussion_r162154476>:
>
> > @@ -31,6 +32,8 @@
>      background-color: $grey-000
>      text-align: left
>      width: 100%
> +    //height: -webkit-fill-available
> +    height: calc(100% - 50px);
>
> Yeah def, I realized that too and was working on it locally. I had tried
> setting the bottom to a standard 50px, but that doesn't really work for all
> displays. I think changing the actual HTML structure of the template like
> you mentioned might be worth it here. Trying to get adjacent anchor tags to
> fill up the available space while pass percy is proving difficult.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/CityOfBoston/patterns/pull/211#discussion_r162154476>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAP65YoQ9R8GM-Smva4C_MNHlV2IdHauks5tLkm6gaJpZM4RSLwX>
> .
>



-- 

[image: COB_B_Blue_square-01.png]

Fin Hopkins

Software Engineer, Digital Team

Department of Innovation and Technology (703)

fin.hopkins@boston.gov

they / them / theirs
